config.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CONFIG_USER_H
```

Comment:
This looks to be an extra file, or in the wrong place.  

Could you remove this file?